data/json/item_groups.json

```diff
@@ -9387,5 +9387,81 @@
       ["boxer_shorts", 30],
       ["briefs", 30]
     ]
+  },{
+    "type" : "item_group",
+    "id" : "army_personal_locker",
+    "items" : [
+      ["jumpsuit", 40],
+      ["pants_army", 70],
+      ["jacket_army", 70],
+      ["socks", 40],
+      ["winter_pants_army", 20],
+      ["winter_jacket_army", 20],
+      ["winter_gloves_army", 20],
```

Comment:
Trying to balance it is a bit complicated.  In the real world the army only issues a single set of winter gear in most situations so they would more than likely be wearing it.